### ISSUE TITLE
Pass the Hash/delp

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuza55/opendlp#12]
```
What steps will reproduce the problem?
1.OpenDLP functions properly and deploys agents with password 
2.Fails with error Filesys::smbclient::_write: fd is not of type SMBCFILEPtr at 
/usr/lib/Perl5/Filesys/SmbClient.pm line 347
when starting scan with passthehash
3.Pass the hash functions properly when done manually

Where should I look to narrow down the problem

```

Original issue reported on code.google.com by `f8ler...@gmail.com` on 11 Feb 2011 at 3:13

**Comment by GoogleCodeExporter** [comment]
```
I didn't test the patch posted on Twitter, I just rolled it into OpenDLP (with 
one modification to an INSERT statement).  What if you roll back to OpenDLP 
0.2.5 and patch it with this: http://raptor-jes.us/pth_mod.patch
```

Original comment by `andrew.O...@gmail.com` on 11 Feb 2011 at 5:36

**Comment by GoogleCodeExporter** [comment]
```
I rolled back to 2.5 and applied the patch. DBD::mysql::st execute failed: 
called with 21 bind variables when 22 are needed at 
/var/www/OpenDLP/web/bin/profiles-new.html line 413. I fixed that by adding the 
hash=? where the insert is. However it still fails with the same error as i 
first stated when i start a scan with the hash. 
```

Original comment by `f8ler...@gmail.com` on 11 Feb 2011 at 6:31

**Comment by GoogleCodeExporter** [comment]
```
I would suggest contacting the author of the patch first. However, when I get 
more time, I will look into the problem and see what's going on (this may not 
be for a few more weeks though). Thanks for reporting this bug.
```

Original comment by `andrew.O...@gmail.com` on 11 Feb 2011 at 6:52

**Comment by GoogleCodeExporter** [comment]
```
Will do although i am still not above me just screwing things up :)
```

Original comment by `f8ler...@gmail.com` on 11 Feb 2011 at 9:24

**Comment by GoogleCodeExporter** [comment]
```
I released a new VirtualBox VM based on Ubuntu 11.04 and OpenDLP 0.4.1. This 
new version has fully working pass-the-hash support for agent and agentless 
scans. Please try this VM if you are still having issues getting pass-the-hash 
working, thanks.
```

Original comment by `andrew.O...@gmail.com` on 15 Aug 2011 at 6:02
- Changed state: **Fixed**
